alienfile

```diff
@@ -3,10 +3,26 @@
 use alienfile;
 
 plugin 'PkgConfig' => (
-    pkg_name        => [qw/ tidy tidy5 tidy-html5 /],
+    pkg_name        => 'tidy',
     minimum_version => '5.6.0',
 );
 
+meta->around_hook('probe' => sub {
+    my($orig, $build) = @_;
+
+    my $type = $orig->($build);
+    if($type eq 'system') {
+        foreach my $exe (qw( tidy5 tidy )) {
+            if(`$exe -version` =~ /HTML Tidy /) {
+                $build->runtime_prop->{command} = $exe;
+                return 'system';
+            }
+        }
+    }
+
+    return 'share';
```

Comment:
if we detect a system install (via pkg-config), but can't find an executable, then we assume that the system install is incomplete and instead attempt a share install.  if we don't do this then the test will fail for incomplete system packages.